jax/_src/lax/parallel.py

```diff
@@ -252,13 +252,17 @@ def pswapaxes(x, axis_name, axis):
       :func:`jax.pmap` documentation for more details).
     axis: int indicating the unmapped axis of ``x`` to map with the name
       ``axis_name``.
+    axis_index_groups: optional list of lists containing axis indices (e.g. for
```

Comment:
Can you please update the docstring above? The property that the axis size must be equal to the size of the mapped axis is not true. It should be equal to the group in which the collective is performed, which just happen to encompass all devices by default. I assume that the same change has to happen in the docstring of `all_to_all`

Reply:
Done.